### PR TITLE
PSADT 3.10.2

### LINKED ISSUE #950: Align AppDeployToolkitConfig.xml with 3.10 release of PSADT

### LINKED ISSUE #990: Extend UI languages and fix spelling mistakes
In the ADTConfig.xml there are spelling mistakes such as "Defferals". Fix these.
Also import languages from new PSADT config and extend neo language prompts

### LINKED ISSUE #1172: Text in AppDeployToolkitConfig.xml are wrong Bug
### Confirmation

- [X] My issue is not related to the behaviour of a package or product provided by neo42
- [X] I understand that issues created in this repository are not subject to any support agreements made with neo42

### Description

Correct the following text:
<NxtWelcomePrompt_ApplicationClose>The following programs must be closed before the {0} can **procced.**</NxtWelcomePrompt_ApplicationClose>

Correct are:
<NxtWelcomePrompt_ApplicationClose>The following programs must be closed before the {0} can proceed.</NxtWelcomePrompt_ApplicationClose>

### Release

2024.11.27.65

### Windows build

_No response_

### Architecture

x64, x86

### Deployment System

Matrix42 Empirum (incl. Cloud), Microsoft Intune, Omnissa Workspace One, Microsoft Configuration Manager

### Relevant logs

_No response_

### PR DESCRIPTION
**Pull request by neo42JBR**
Import new and updated values from 3.10.1 release of PSADT.
Sensible defaults for new values have been selected.

Values can be adjusted in downstream PRs

### CODE REVIEW

File: AppDeployToolkit/AppDeployToolkitConfig.xml [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+	WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+	PARTICULAR PURPOSE. See the GNU General Public License
+	for more details. You should have received a copy of the GNU Lesser General Public License along
+	with this program. If not, see <http://www.gnu.org/licenses/>.
+	-->
+
+	<!--Configuration
+	File Details-->
+	<Config_File>
+		<Config_Version>3.10.1</Config_Version>
```
Comment (neo42mgn) [inline AppDeployToolkit/AppDeployToolkitConfig.xml]: 3.10.2

File: AppDeployToolkit/AppDeployToolkitConfig.xml [hunk 2]
```diff
@@ -460,946 +543,969 @@
...
+		<RestartPrompt_MessageTime>Salvare il lavoro e riavviare entro il tempo assegnato.</RestartPrompt_MessageTime>
+		<RestartPrompt_MessageRestart>Il computer verrà riavviato automaticamente al termine del conto alla rovescia.</RestartPrompt_MessageRestart>
+		<RestartPrompt_TimeRemaining>Tempo rimanente:</RestartPrompt_TimeRemaining>
+		<RestartPrompt_ButtonRestartLater>Minimizzare</RestartPrompt_ButtonRestartLater>
+		<RestartPrompt_ButtonRestartNow>Riavvia Ora</RestartPrompt_ButtonRestartNow>
+
+		<DeploymentType_InstallVerb>installato</DeploymentType_InstallVerb>
+		<DeploymentType_UnInstallVerb>disinstallato</DeploymentType_UnInstallVerb>
+		<DeploymentType_RepairVerb>riparato</DeploymentType_RepairVerb>
+		<NxtWelcomePrompt_FollowApplication>La seguente applicazione sta per essere {0}:</NxtWelcomePrompt_FollowApplication>
```
Comment (neo42mgn) [inline AppDeployToolkit/AppDeployToolkitConfig.xml]: Move all the translations to a different issue, branch and PR, only en and de in this PR

File: AppDeployToolkit/AppDeployToolkitConfig.xml [hunk 3]
```diff
@@ -157,137 +190,153 @@
...
+		<NxtWelcomePrompt_ApplicationClose>The following applications must be closed before the {0} can proceed.</NxtWelcomePrompt_ApplicationClose>
 		<NxtWelcomePrompt_SaveWorkWithoutCloseButton>Please save your work and close the applications.</NxtWelcomePrompt_SaveWorkWithoutCloseButton>
 		<NxtWelcomePrompt_SaveWork>Please save your work, close the applications and this process will be continued. Alternativly, save your work and click 'Close applications'.</NxtWelcomePrompt_SaveWork>
 		<NxtWelcomePrompt_ApplicationName>Application name</NxtWelcomePrompt_ApplicationName>
 		<NxtWelcomePrompt_StartedBy>Started by</NxtWelcomePrompt_StartedBy>
 		<NxtWelcomePrompt_ChooseDefer>You can choose to defer the {0} until the deferral expires:</NxtWelcomePrompt_ChooseDefer>
-		<NxtWelcomePrompt_RemainingDefferals>Remaining deferrals: {0}</NxtWelcomePrompt_RemainingDefferals>
+		<NxtWelcomePrompt_RemainingDeferals>Remaining deferrals: {0}</NxtWelcomePrompt_RemainingDeferals>
 		<NxtWelcomePrompt_DeferalExpired>Once the defferal is expired, you will no longer have the option to defer.</NxtWelcomePrompt_DeferalExpired>
```
Comment (neo42mgn) [inline AppDeployToolkit/AppDeployToolkitConfig.xml]: also here: defferal -> deferal

**Review by neo42mgn — CHANGES_REQUESTED** [review]

File: AppDeployToolkit/AppDeployToolkitConfig.xml [hunk 4]
```diff
@@ -157,1249 +190,1245 @@
...
+			Por favor, libere suficiente espacio en disco para continuar con la instalación.</DiskSpace_Message>
+		<ClosePrompt_ButtonContinue>Continuar</ClosePrompt_ButtonContinue>
+		<ClosePrompt_ButtonContinueTooltip>Solo seleccione "Continuar" después de cerrar la(s) aplicacion(es) de la lista.</ClosePrompt_ButtonContinueTooltip>
+		<ClosePrompt_ButtonClose>Cerrar Programas</ClosePrompt_ButtonClose>
+		<ClosePrompt_ButtonDefer>Aplazar</ClosePrompt_ButtonDefer>
+		<ClosePrompt_Message>Los siguientes programas deben estar cerrados antes de que la instalación pueda continuar.
+
+			Guarde su trabajo, cierre los programas y luego continúe.
+			Alternativamente, guarde su trabajo y haga clic en "Cerrar programas".</ClosePrompt_Message>
+        <closePrompt_CountdownMessage>NOTA: El/los programa(s) se cerrará(n) automáticamente en:</closePrompt_CountdownMessage>
```
Comment (neo42mgn) [inline AppDeployToolkit/AppDeployToolkitConfig.xml]: use tab instead of 4 whitespaces and start with capital letter

**Review by neo42mgn — CHANGES_REQUESTED** [review]

File: AppDeployToolkit/AppDeployToolkitConfig.xml (hunk 4 above)
Comment (neo42JBR) [reply AppDeployToolkit/AppDeployToolkitConfig.xml]: Was in upstream. Fixed anyway

**Review by neo42mgn — APPROVED** [review]

**Pull request merged**